File: CrmApp.Domain/Entities/ContractorContacts.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CrmApp.Domain.Entities
{
    public class ContractorContacts
    {
        [Key]
        public int Id { get; set; }
        public int ContractorId { get; set; }
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public string DisplayName { get; set; } = string.Empty;
        public string? Position { get; set; }
        public string? Email { get; set; }
        public string? PhoneNumber { get; set; }
        public DateTime CreatedAt { get; set; }
        public int CreatedBy { get; set; }
    }
}

File: CrmApp.Client/src/models/ContractorContacts.ts
export class ContractorContacts {
    
    constructor() {
        this.isEditMode = true;
    }

    id: number = 0;
    contractorId: number = 0;
    firstname: string = "";
    lastname: string = "";
    displayName: string = "";
    position?: string = "";
    email?: string = "";
    phoneNumber?: string = "";
    createdAt: Date = new Date();
    createdBy: number = 0;

    isEditMode: boolean = false;
}